src/imagery/i.sentinel/i.sentinel.parallel.download/i.sentinel.parallel.download.py

```diff
@@ -164,10 +165,13 @@ def scenename_split(scenename, datasource, esa_name_for_usgs=False):
         end_day_dt = dt_obj + timedelta(days=1)
         start_day = start_day_dt.strftime("%Y-%m-%d")
         end_day = end_day_dt.strftime("%Y-%m-%d")
-        # get query string
-        if not scenename.endswith(".SAFE"):
-            scenename = scenename + ".SAFE"
-        query_string = "filename={}".format(scenename)
+        if datasource == "GCS":
+            query_string = f"identifier={scenename.replace('.SAFE', '')}"
```

Comment:
```suggestion
            query_string = f"identifier={scenename.removesuffix('.SAFE')}"
```
Here you remove the suffix `.SAFE` ...

Reply:
`removesuffix` was introduced in Python 3.9 - not sure if we can expect this?

Reply:
Indeed, I missed this. I would not expect/require Python 3.9, forget about the suggestion.